pyEpiabm/pyEpiabm/sweep/host_progression_sweep.py

```diff
@@ -281,19 +283,117 @@ def __call__(self, time: float):
             Current simulation time
 
         """
+        asympt_uninf = []
         for cell in self._population.cells:
             for person in cell.persons:
                 if person.time_of_status_change is None:
                     assert person.infection_status \
                                     in [InfectionStatus.Susceptible]
```

Comment:
This is an odd construction - do you just mean the person is susceptible?

Reply:
This wasn't added by me but I think it makes more sense to use to is.susceptible() function.

Reply:
Yes, or even just that  `person.infection_status == InfectionStatus.Susceptible`